08-transport.Rmd

```diff
@@ -558,31 +623,32 @@ m_leaflet = qtm(ways, lines.col = "highway", lines.lwd = 3, lines.palette = c("l
 tmap::tmap_leaflet(m_leaflet)
 ```
 
+<!-- Maybe we should mention and/or justify why we think that the route between polygon centroids is probably also the most used one. What if most travellers travel from the northwestern part of polygon A to the northeastern part of polygon B?  -->
```

Comment:
Maybe we should mention and/or justify why we think that the route between two polygon centroids is probably also the most used one. What if most travellers travel from the northwestern part of polygon A to the northeastern part of polygon B?

Reply:
Sorted